internal/common/voting/rulevote.go

```diff
@@ -6,46 +6,56 @@ import (
 )
 
 type RuleVote struct {
+	//Checked by RuleVote
 	ruleToVote    string
 	islandsToVote []shared.ClientID
-	votes         []bool
+	//Held by RuleVote
+	ballots []bool
 }
 
-type RuleVoteResult struct {
-	votesInFavour uint
-	votesAgainst  uint
-	Result        bool
+type BallotBox struct {
+	VotesInFavour uint
+	VotesAgainst  uint
```

Comment:
`TotalVotes` and `VotesInFavour`?

Reply:
Votes for and against work better for our CountVotesMajority() function